resources/sts/4.14/hypershift/sts_extended_hcp_support_permission_policy.json

```diff
@@ -62,49 +137,300 @@
+                "StringEquals": {
+                    "aws:ResourceTag/red-hat-managed": "true"
+                }
+            }
+        },
+        {
+            "Sid": "Ip",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeCoipPools",
+                "ec2:DescribeIpv6Pools",
+                "ec2:DescribePublicIpv4Pools",
+                "ec2:GetAssociatedIpv6PoolCidrs"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "InternetGateway",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeEgressOnlyInternetGateways",
+                "ec2:DescribeInternetGateways"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "Describetag",
+            "Effect": "Allow",
+            "Action": [
+               "ec2:DescribeTags" 
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "PrefixLists",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribePrefixLists",
+                "ec2:DescribeManagedPrefixLists"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "PrefixListsentries",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:GetManagedPrefixListEntries"
+            ],
+            "Resource": "arn:aws:ec2:*:*:prefix-list/*"
+        },
+        {
+            "Sid": "Image",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeImages"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "ImagaeAttibute",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeImageAttribute"
+            ],
+            "Resource": "arn:aws:ec2:*:*:image/*"
+        },
+        {
+            "Sid": "IdFormat",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribePrincipalIdFormat",
+                "ec2:DescribeIdentityIdFormat",
+                "ec2:DescribeIdFormat",
+                "ec2:DescribeAggregateIdFormat"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "AddressAttribute",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeAddressesAttribute"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "Addresses",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeAddresses"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "AccountAttributes",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeAccountAttributes"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "ByoipCidrs",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeByoipCidrs"
```

Comment:
What is this permission used for?

Reply:
I'm not sure if it's worth keeping this permission or not, as it could potentially be useful while troubleshooting networking